### PR TITLE
do_command added to recompute / reload embeddings

### PR DESCRIPTION
**Pull request by felixreichenbach**
When new face images / folders are added the module needed to be restarted. I have added a do_command which will set the embeddings to null and then recomputes them.

### CODE REVIEW

File: src/face_detection_module.py [hunk 1]
```diff
@@ -138,4 +138,9 @@ async def do_command(self,
                         *,
                         timeout: Optional[float] = None,
                         **kwargs):
-        raise NotImplementedError
\ No newline at end of file
+        if command["command"] == "recompute_embeddings" :
+            self.identifier.known_embeddings = dict()
+            self.identifier.compute_known_embeddings()
+            LOGGER.error("Recompute Embeddings!")
```
Comment (kharijarrett) [inline src/face_detection_module.py]: Being nosy here but do we want this to be a LOGGER.info() instead?

Comment (kharijarrett) [reply src/face_detection_module.py]: The difference being it doesn't show as if you messed something up

Comment (felixreichenbach) [reply src/face_detection_module.py]: you are absolutely right :-)

**Comment by Rob1in** [comment]
lgtm! I can do the release when it's merged so it will be uploaded on the registry

**Review by Rob1in — APPROVED** [review]

**Pull request merged**